src/components/shared/Entity/EndpointConfig/index.tsx

```diff
@@ -97,6 +103,16 @@ function EndpointConfig({
 
     const forceClose = !editWorkflow && draftId !== null;
 
+    useEffect(() => {
+        if (connectorTag) {
+            setDocsURL(connectorTag.documentation_url);
+        }
+
+        return () => {
+            resetState();
+        };
+    }, [connectorTag, resetState, setDocsURL]);
```

Comment:
This might not be the best spot in the long run... but it works for now. This is previously the component that passed the `documentation_url`  to the header so it seemed to make sense.